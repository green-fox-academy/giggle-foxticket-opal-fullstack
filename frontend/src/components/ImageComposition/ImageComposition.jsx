import React from 'react';
import cat from '../../assets/images/card-cat.jpeg';
import './ImageComposition.styles.sass';
import Button from '../Button/Button';

const ImageComposition = () => {
  return (
    <>
      <section className="container">
        <div className="left-half">
          <article>
            <h1>Super Size Me</h1>
            <p>
              Director Morgan Spurlocks social experiment in fast-food
              gastronomy sees him attempting to subsist uniquely on food from
              the McDonalds menu for an entire month. In the process his weight
              balloons, his energy level plummets and he experiences all sorts
              of unexpected -- and terrifying -- sid e effects. H e also
              examines the corporate giants growing role in the lives of
              American consumers and explores its methods of indoctrinating
              young people and its contribution to Americas obesity epidemic.
            </p>
          </article>
          <Button>Learn More</Button>
        </div>
        <div className="right-half">
          <article>
            <img src={cat} alt="hello" className="cat" />
          </article>
        </div>
      </section>
    </>
  );
};

export default ImageComposition;
