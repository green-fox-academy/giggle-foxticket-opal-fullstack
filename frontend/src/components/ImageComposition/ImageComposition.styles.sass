article
  position: relative
  text-align: center
  padding: 1rem

.container
  display: grid
  grid-template-columns: auto auto auto

  .left-half
    grid-column: 1 / 2
    h1, p
      text-align: left
    button
      margin-right: 1rem
      float: right

  .right-half
    grid-column: 2 / 3
    .cat
      border-radius: 10px
      width: 800px



