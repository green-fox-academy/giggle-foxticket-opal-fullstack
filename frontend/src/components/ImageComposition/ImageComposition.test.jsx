import React from 'react';
import ImageComposition from './ImageComposition';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<ImageComposition />).toJSON();
  expect(tree).toMatchSnapshot();
});
