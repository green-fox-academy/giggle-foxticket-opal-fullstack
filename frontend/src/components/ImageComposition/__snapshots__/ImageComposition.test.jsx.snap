// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="container"
>
  <div
    className="left-half"
  >
    <article>
      <h1>
        Super Size Me
      </h1>
      <p>
        Director Morgan Spurlock's social experiment in fast-food gastronomy sees him attempting to subsist uniquely on food from the McDonald's menu for an entire month. In the process his weight balloons, his energy level plummets and he experiences all sorts of unexpected -- and terrifying -- side effects. He also examines the corporate giant's growing role in the lives of American consumers and explores its methods of indoctrinating young people and its contribution to America's obesity epidemic.
      </p>
    </article>
    <button>
      Learn More
    </button>
  </div>
  <div
    className="right-half"
  >
    <article>
      <img
        alt="hello"
        className="cat"
        src="card-cat.jpeg"
      />
    </article>
  </div>
</section>
`;
